tests/test_ssl.py

```diff
@@ -423,6 +423,24 @@ def test_set_cipher_list_wrong_args(self, context, cipher_list, error):
         with pytest.raises(error):
             context.set_cipher_list(cipher_list)
 
+    def test_set_cipher_list_no_cipher_match(self, context):
```

Comment:
Instead of a new test, can you just split `test_set_cipher_list_wrong_args` into two, since this really duplicates half of that.